engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorSpec.scala

```diff
@@ -59,7 +63,7 @@ class WorkflowExecutionActorSpec extends CromwellTestkitSpec with BeforeAndAfter
       val isFatal = (_: Throwable) => false
 
       // Sleep a bit, and let the metadata do its thing in the background.
-      Thread.sleep(1000L)
+      Thread.sleep(3.seconds.dilated.toMillis)
```

Comment:
ToL: I cry a little when we just "sleep and hope" rather than "await expectation for a max of 3 seconds"